apps/web/src/messages/experienceMessages.ts

```diff
@@ -0,0 +1,56 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  award: {
+    defaultMessage: "Awards and recognition",
+    id: "DRYl88",
+    description: "Title for award experience section",
+  },
+  community: {
+    defaultMessage: "Community participation",
+    id: "Uy5Dg2",
+    description: "Title for community experience section",
+  },
+  education: {
+    defaultMessage: "Education and certificates",
+    id: "PFoM2I",
+    description: "Title for education experience section",
+  },
+  personal: {
+    defaultMessage: "Personal learning",
+    id: "+AUFyC",
+    description: "Title for personal learning section",
+  },
+  work: {
+    defaultMessage: "Work experience",
+    id: "giUfys",
+    description: "Title for work experience section",
+  },
+  unknown: {
+    defaultMessage: "Unknown Experience",
+    id: "U/Lv8i",
+    description: "Title for unknown experiences",
+  },
+  awardIssuedBy: {
+    defaultMessage: "{title} issued by {issuedBy}",
+    id: "4BpFoX",
+    description: "The award title is issued by some group",
+  },
+  communityAt: {
+    defaultMessage: "{title} at {organization}",
+    id: "vV0SDz",
+    description: "Title at organization",
+  },
+  educationAt: {
+    defaultMessage: "{areaOfStudy} at {institution}",
```

Comment:
The title is broken into "degree" (bolded) "from" "institution"

Reply:
Oh sorry I should of been a little clearer. The text should read `{areaOfStudy} from {institution}` not {areaOfStudy} at {institution}.

Reply:
Oh sorry I should of been a little clearer. The text should read {areaOfStudy} from {institution} not {areaOfStudy} at {institution}.

Reply:
Hmm, that would read very odd even if we start with the type of education🤔 For example, a user could select "Online course" or "Postdoctoral Fellowship" then it would read:

"Online course from University" which does not make much sense. 😅